lib/console.js

```diff
@@ -33,7 +35,8 @@ function Console(stdout, stderr) {
 }
 
 Console.prototype.log = function() {
-  this._stdout.write(util.format.apply(this, arguments) + '\n');
+  this._stdout.write(util.format.apply(this, arguments).replace(/^/gm,
+          Array(this._group * 2 + 1).join(' ')) + '\n');
```

Comment:
This jsperf is using a shim for String.prototype.repeat. [Native is faster](http://jsperf.com/faster-string-repeat/16)
Anyway both show that Array#join is the slowest of all